README.rst

```diff
@@ -35,13 +35,12 @@ For that, it gives you a class decorator and a way to declaratively define the a
 
 .. code-block:: pycon
 
-   >>> from typing import List
-   >>> import attr
+   >>> from attr import define, Factory
 
-   >>> @attr.define
+   >>> @define
    ... class SomeClass:
    ...     a_number: int = 42
-   ...     list_of_numbers: List[int] = attr.Factory(list)
+   ...     list_of_numbers: list[int] = Factory(list)
```

Comment:
I'm nervous about `list[int]` because that only works in 3.9+ and I can imagine lots of people in a lower version.  Though I'm not sure.  I could've sworn there was a chart of which versions people run but I can't find it now.

Reply:
I tried `list` but we can't because RTD is still on old Python and doctest doesn't support future imports. :(

Reply:
(I have just moved `main` to 3.8, but it's still not gonna work. I'm afraid we'd need 3.10)

Reply:
Aight, we can move back to `List` for the time being.